packages/caldav/README.md

```diff
@@ -0,0 +1,33 @@
+# timeld - CalDAV integration
+
+This packages supports live update of timesheet entries:
+- _from_ a CalDAV end-point
+- _to_ **timeld**.
+
+## installation & configuration
+
+This integration package is included in the gateway.
+
+The integration requires the following configuration:
+- `uri`: calendar URI (https)
+- `auth.user`: CalDAV account user (optional if calendar is public)
+- `auth.pass`: CalDAV account password (optional if calendar is public)
+- `owner`: calendar owner IRI (optional; defaults to the timesheet owner account)
+- `pollInterval` CalDAV polling interval, in millis (optional; defaults to 10000)
+
+Configuration defaults may already be included in the gateway. Timesheet-specific configuration can also be included when adding the integration, see below.
```

Comment:
1. Ditto re: case of "Gateway".
2. As per conversation, replace "integration" with "connector".